runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/action.json

```diff
@@ -15,6 +15,7 @@
     "Action.Details.Form": "Form",
     "Action.Details.Form.Script.Error": "Required. Must start with external.",
     "Action.Details.Form.Label.Error": "Required",
+    "Action.Details.Form.Validation.Error": "Invalid form values",
```

Comment:
Is this error content enough to explain what's the issue?

Reply:
Should be, as this is only when you click on submit and you can see on the form what is the issue :)